src/api/index.js

```diff
@@ -0,0 +1,23 @@
+import { PROTOCOLS_API, PROTOCOL_API } from 'constants/index'
+import { standardizeProtocolName } from 'utils'
+
+export const getProtocols = () =>
+  fetch(PROTOCOLS_API)
+    .then(r => r.json())
+    .then(({ protocols, chains, protocolCategories }) => ({
+      protocolsDict: protocols.reduce((acc, curr) => {
+        acc[standardizeProtocolName(curr.name)] = curr
```

Comment:
shouldn't we run this on the server?

Reply:
I think it is since I call it in getStaticProps, or we could just add it to the lambda, but not too sure how big we want the payload